### PR TITLE
Feature/issue 679 update usfm linter output

### PR DESCRIPTION
**Pull request by bspidel**
changes to make error logs look like requirement in issue. Most logs now look like

\<book\>[ \<ch\> [ :\<verse\>]] - \<error text\>

### CODE REVIEW

File: libraries/linters/usfm_linter.py [hunk 1]
```diff
@@ -82,9 +82,9 @@ def parse_usfm_text(self, sub_path, file_name, book_text, book_full_name, book_c
                 self.found_books.append(book_code)
 
             if len(errors):
-                prefix = "File '{0}' book code '{1}': ".format(sub_path, book_code)
                 for error in errors:
-                    self.log.warning(prefix + error)
+                    parts = error.split( ":" )
+                    self.log.warning( "{0} {1} - {2}".format(book_code, " ".join( parts[1:] ), parts[0] ))
```
Comment (PhotoNomad0) [inline libraries/linters/usfm_linter.py]: should have a sanity check here in case there is no ':' in the error message.

**Review by PhotoNomad0 — CHANGES_REQUESTED** [review]

**Comment by PhotoNomad0** [comment]
@bspidel - do you have an example of the updated output?

Also should put a link to the issue in the PR description.

File: libraries/linters/usfm_linter.py (hunk 1 above)
Comment (richmahn) [reply libraries/linters/usfm_linter.py]: Also, my IDE linter always complains about these spaces after the parentheses. Is it possible to remove the spaces? I know, a bit picky, but really prefer the code to be the same as above and below.

**Review by richmahn — CHANGES_REQUESTED** [review]
As with all our stuff, needs a unit test that will verify that the output of the linter, the warnings, are as we want them to be.

**Review by PhotoNomad0 — APPROVED** [review]

**Review by richmahn — APPROVED** [review]

**Pull request merged**